front/app/components/UI/GanttChart/components/GanttItem.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+
+import { Box, Tooltip, colors } from '@citizenlab/cl2-component-library';
+import { max, min } from 'date-fns';
+
+import { rowHeight } from '../utils';
+
+import GanttItemTextLabel from './GanttItemTextLabel';
+
+import type { GanttItem, ViewBounds } from '../types';
+
+interface GanttItemProps {
+  item: GanttItem;
+  index: number;
+  viewBounds: ViewBounds;
+  getOffset: (date: Date) => number;
+  getDuration: (start: Date, end: Date) => number;
+  unitWidth: number;
+  renderItemTooltip?: (item: GanttItem) => React.ReactNode;
+}
+
+const GanttItem = ({
+  item,
+  index,
+  viewBounds,
+  getOffset,
+  getDuration,
+  unitWidth,
+  renderItemTooltip,
+}: GanttItemProps) => {
+  const start = item.start ? new Date(item.start) : undefined;
```

Comment:
It's tricky to test right now because I am in Europe, but generally doing simply `new Date` on a date that comes from our backend causes problems in e.g. Brazil, because it will interpret the date as midnight in GMT (which means that in Brazil it will be one day earlier). So I would instead recommend to use the `parseBackendDateString` function defined in `front/app/utils/dateUtils.ts`.